src/style/index.css

```diff
@@ -18,4 +18,11 @@
 
 mjx-container.MathJax[jax='CHTML'][display='true'] {
   display: inline-block !important;
+  overflow-x: auto;
+  overflow-y: hidden;
+  padding: 0.5em;
+  margin: 0;
```

Comment:
To formler som havner over hverandre treffer kant til kant når margin er 0. Fjerne denne eller øke litt?

Ellers ser det bra ut!